resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfiguration.java

```diff
@@ -90,19 +85,29 @@ public ReactorCircuitBreakerAspectExt reactorCircuitBreakerAspect() {
 	}
 
 	/**
-	 * Initializes the backends configured in the properties.
+	 * Initializes a circuitBreaker registry.
 	 *
-	 * @param circuitBreakerRegistry The circuit breaker registry.
+	 * @param circuitBreakerProperties The circuit breaker configuration properties.
+	 *
+	 * @return a CircuitBreakerRegistry
 	 */
-	public void initializeBackends(CircuitBreakerRegistry circuitBreakerRegistry) {
+	public CircuitBreakerRegistry createCircuitBreakerRegistry(CircuitBreakerConfigurationProperties circuitBreakerProperties) {
+		Map<String, CircuitBreakerConfig> configs = circuitBreakerProperties.getConfigs()
+				.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey,
+						entry -> circuitBreakerProperties.createCircuitBreakerConfig(entry.getValue())));
+
+		return CircuitBreakerRegistry.of(configs);
+	}
 
+	/**
+	 * Initializes the CircuitBreaker registry.
+	 *
+	 * @param circuitBreakerRegistry The circuit breaker registry.
+	 */
+	public void initCircuitBreakerRegistry(CircuitBreakerRegistry circuitBreakerRegistry) {
```

Comment:
we can not the back-ends the same away as above in  CircuitBreakerRegistry bean init ?